### PR TITLE
14.0 mig web listview range select

### PR DESCRIPTION
**Pull request by Kev-Roche**

### CODE REVIEW

**Comment by emagdalenaC2i** [comment]
@Kev-Roche LGTM But you didn' follow the migration guidelines 
See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

File: web_listview_range_select/static/src/js/web_listview_range_select.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    "use strict";
+
+    var ListRenderer = require("web.ListRenderer");
+
+    ListRenderer.include({
+        /*
+        I extend 'events' because in v13 Odoo catches 'change' event instead
+        of the 'click' event for the selector .o_list_record_selector ",
+        so shift + [click] is not caught.
+        https://github.com/OCA/OCB/blob/13.0/addons/web/static/src/js/views/list/list_renderer.js#L42:L42
```
Comment (BT-aestebanez) [inline web_listview_range_select/static/src/js/web_listview_range_select.js]: This comment is deprecated, and should change its reference to v14

**Review by BT-aestebanez — APPROVED** [review]
LGTM but, can update the comment of the JS file?
In addition, the links in the readme file point to 13.0

File: web_listview_range_select/static/src/js/web_listview_range_select.js (hunk 1 above)
Comment (Kev-Roche) [reply web_listview_range_select/static/src/js/web_listview_range_select.js]: Can I fully remove this comment?

Comment (BT-aestebanez) [reply web_listview_range_select/static/src/js/web_listview_range_select.js]: In my opinion, that would be the best approach :)

Comment (Kev-Roche) [reply web_listview_range_select/static/src/js/web_listview_range_select.js]: done !

Comment (BT-aestebanez) [reply web_listview_range_select/static/src/js/web_listview_range_select.js]: Thanks, please update the links of the readme file :)

Comment (Kev-Roche) [reply web_listview_range_select/static/src/js/web_listview_range_select.js]: oops, my bad, also done

**Review by BT-aestebanez — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Review by britoederr — APPROVED** [review]

**Comment by BT-aestebanez** [comment]
@simahawk can this be merged?

**Review by andreampiovesana — APPROVED** [review]
tested ok 
go

**Comment by BT-fgarbely** [comment]
Hello @Yajo @pedrobaeza @gurneyalex 

Does anyone know the current state here?
Can this be merged?

Thanks in advance

**Pull request merged**